Selector/example.jsx

```diff
@@ -55,6 +57,46 @@ import UncontrolledSelector from '@klarna/ui/uncontrolled/Selector'`,
           onSelect={(v) => (v)}
         />
       }
+    },
+
+    {
+      title: 'Selector Input',
+      require: `import Fieldset from '@klarna/ui/Fieldset'
+import * as Selector from '@klarna/ui/Selector'`,
+      type: LIVE,
+
+      examples: {
+        Variations: <Fieldset margins>
+          <Selector.Input
+            onClick={(e) => { console.info('it was clicked', e) }}
```

Comment:
Changing this to `onClick={(e) => console.info('it was clicked', e)}` makes it render more pretty in the JSX output. I know it's misleading because `console.info` is not actually returning anything, this is just to make it more concise in the example.